hyprtester/test.conf

```diff
@@ -334,3 +334,7 @@ gesture = 3, horizontal, mod:ALT, workspace
 
 gesture = 4, up, dispatcher, sendshortcut, ctrl, d, activewindow
 
+# testTags
+windowrule = tag +testTag, class:tagged
+windowrule = noshadow, tag:negative:testTag
+windowrule = noborder, tag:testTag
```

Comment:
dispatch this dynamically with `getFromSocket("/keyword ...")` to not potentially taint other tests (`negative:` will target all windows from other tests)

At the end of your test fn do `/reload` to reload the cfg